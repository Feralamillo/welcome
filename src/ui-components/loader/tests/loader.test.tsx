import { render } from '@testing-library/react';

import { Loader } from '../loader';

describe('<Loader />', () => {
  it('should render the component', () => {
    const { container } = render(<Loader />);

    expect(container).toMatchSnapshot();
  });
});
