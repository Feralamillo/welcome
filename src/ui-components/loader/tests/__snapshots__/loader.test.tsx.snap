// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should render the component 1`] = `
<div>
  <div
    class="loader"
  />
</div>
`;
